docs/topics/auth/passwords.txt

```diff
@@ -172,34 +172,50 @@ iterations needs to be increased. We've chosen a reasonable default (and will
 increase it with each release of Django), but you may wish to tune it up or
 down, depending on your security needs and available processing power. To do so,
 you'll subclass the appropriate algorithm and override the ``iterations``
-parameters. For example, to increase the number of iterations used by the
```

Comment:
IMO we don't need a full example, it should be enough to mention a proper parameter for `bcrypt` algorithm, e.g.
```diff
diff --git a/docs/topics/auth/passwords.txt b/docs/topics/auth/passwords.txt
index 4b13334379..363c90d6fc 100644
--- a/docs/topics/auth/passwords.txt
+++ b/docs/topics/auth/passwords.txt
@@ -172,8 +172,9 @@ iterations needs to be increased. We've chosen a reasonable default (and will
 increase it with each release of Django), but you may wish to tune it up or
 down, depending on your security needs and available processing power. To do so,
 you'll subclass the appropriate algorithm and override the ``iterations``
-parameters. For example, to increase the number of iterations used by the
-default PBKDF2 algorithm:
+parameter (use the ``rounds`` parameter when subclassing
+``BCryptPasswordHasher``). For example, to increase the number of iterations
+used by the default PBKDF2 algorithm:
 
 #. Create a subclass of ``django.contrib.auth.hashers.PBKDF2PasswordHasher``::
 
```

Reply:
Great! I removed the bcrypt example. Instead of "use the rounds parameter when subclassing BCryptPasswordHasher", I wrote "when subclassing a bcrypt hasher", since it'll be the same for any bcrypt based hasher (eg BCryptSHA256PasswordHasher). What do you think?